docs/src/main/asciidoc/logging.adoc

```diff
@@ -9,10 +9,91 @@ include::./attributes.adoc[]
 
 This guide explains logging and how to configure it.
 
+Internally, Quarkus uses JBoss Log Manager and the JBoss Logging facade. +
+You can use the JBoss Logging facade inside your code as it's already provided,
+or any of the supported Logging API listed in the next chapter as Quarkus will send them to JBoss Log Manager.
+
+All the logging configuration will then be done inside your `application.properties`.
+
+== Supported Logging APIs
+
+Applications and components may use any of the following APIs for logging, and the logs will be merged:
+
+* JDK `java.util.logging` (also called JUL)
+* https://github.com/jboss-logging/jboss-logging[JBoss Logging]
+* https://www.slf4j.org/[SLF4J]
+* https://commons.apache.org/proper/commons-logging/[Apache Commons Logging]
+
+Internally Quarkus uses JBoss Logging, you can also use it inside your application so that no other dependencies should be added for your logs.
+
+[source,java]
+----
+import org.jboss.logging.Logger;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class ExampleResource {
+
+    private static final Logger LOG = Logger.getLogger(ExampleResource.class);
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        LOG.info("Hello");
+        return "hello";
+    }
+}
+----
+
+=== What about Apache Log4j 2 ?
+
+link:https://logging.apache.org/log4j/2.x/[Log4j 2] is a logging implementation, it contains a logging backend and a logging facade.
+Quarkus uses the JBoss Log Manager backend, so you will need to include the `log4j2-jboss-logmanager` library to route Log4j 2 logs to JBoss Log Manager.
+
+[source,xml]
+----
+    <dependency>
+      <groupId>org.jboss.logmanager</groupId>
+      <artifactId>log4j2-jboss-logmanager</artifactId>
+    </dependency>
+----
+
+You can then use the Log4J 2 API inside your application.
+
+WARNING: do not include any Log4j 2 dependencies, the `log4j2-jboss-logmanager` library includes what's needed to use Log4j 2 as a logging facade.
+
+NOTE: if you still use the legacy Log4j version 1, you need to use `log4j-jboss-logmanager` instead.
+
+== Logging levels
+
+JBoss logging support the link:https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html[standard JDK log levels] (in descending order):
```

Comment:
These levels should be de-emphasized.  The levels we expect users to use and care about are `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, and `TRACE` - aka the Apache standard levels.  The JUL levels should be the footnote.

Reply:
Just read your detailed explaination on the issues about log levels, I'll update the documentation to emphasize the Apache levels.

Reply:
Done